pipert2/core/base/wrappers/rpc_pipe_wrapper.py

```diff
@@ -0,0 +1,45 @@
+from zerorpc import Server
+from pipert2.core.base.pipe import Pipe
+from pipert2.utils.consts import START_EVENT_NAME, STOP_EVENT_NAME
+
+
+class RPCPipeWrapper(Server):
+    """A RPC wrapper which invokes pipeline cli commands
```

Comment:
Change the way you implement this -> Instead of wrapping the pipe, the pipe will hold it